addons/mail/static/src/models/message/message.js

```diff
@@ -586,9 +586,10 @@ function factory(dependencies) {
             inverse: 'checkedMessages',
             readonly: true,
         }),
-        date: attr({
-            default: moment(),
-        }),
+        /**
+         * Containe the date of the message.
```

Comment:
Typo, also I was hoping the documentation told the type of values here, because it is a bit special (using `moment()`).